package.json

```diff
@@ -153,6 +154,7 @@
     "moment": "^2.24.0",
     "moment-duration-format": "^2.2.1",
     "mz": "^2.6.0",
+    "net-keepalive": "^1.2.1",
```

Comment:
From `https://www.npmjs.com/package/net-keepalive`:
> The Missing (TCP_KEEPINTVL and TCP_KEEPCNT) SO_KEEPALIVE socket option setters and getters for Node using ffi module. Tested on linux, should work on osx and freebsd.

I think it is OK for this feature/fix to apply to Linux only (maybe macOS). The question is: does it raise any errors when ran on Windows? Because raising errors would be a problem. :-)

Reply:
So, in Windows 10, this should work as expected as the `setsockopt()` includes both `TCP_KEEPINTVL` and `TCP_KEEPCNT`. However, in prior versions, these were set as registry keys and will simply be ignored.

I don't think this'll throw an error, but it's probably worth checking.

Reply:
Yeah I initially misread it to support the 3 big ones, testing on a windows vm now.

Reply:
This module isn't supported on windows. Will merge this try to find a method of achieving the same thing on Windows.

Reply:
@CameronDiver, your laptop runs Linux, right? I offer to test it on macOS, just in case. I've tried it now, but it's failing for other reasons. I'll try again when the build tests are passing.